<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>668</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>668</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Icons/pramso_1.ico</normaloff>Icons/pramso_1.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(59, 59, 59);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_63">
         <property name="minimumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 8pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>CLIENTE</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="cliente_edit">
         <property name="minimumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_76">
         <property name="minimumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 8pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>OBSERVACIONES</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="observaciones">
         <property name="minimumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_64">
         <property name="minimumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 8pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>TRACTOR</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="tractor_edit">
         <property name="minimumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_71">
         <property name="minimumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 8pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>RODADO</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="rodado">
         <property name="minimumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_66">
         <property name="minimumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 8pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>COSECHADORA</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="cosechadora_edit">
         <property name="minimumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_73">
         <property name="minimumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 8pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>LOCALIDAD</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="localidad">
         <property name="minimumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_69">
         <property name="minimumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 8pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>PLATAFORMA SOJERA</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="plataforma_sojera">
         <property name="minimumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_74">
         <property name="minimumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 8pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>PROVINCIA</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="provincia">
         <property name="minimumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_70">
         <property name="minimumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 8pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>PLATAFORMA MAICERA</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="plataforma_maicera">
         <property name="minimumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_75">
         <property name="minimumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 8pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>ESTADO</string>
         </property>
        </widget>
        <widget class="QComboBox" name="estado">
         <property name="minimumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <item>
          <property name="text">
           <string>TODOS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NUEVO</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NORMAL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>URGENTE</string>
          </property>
         </item>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_72">
         <property name="minimumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 8pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>CHASIS/EMBOCADOR</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="chasisembocador">
         <property name="minimumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>182</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="buscar">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Buscar</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
